pkg/kubectl/cmd/cmd.go

```diff
@@ -331,7 +332,16 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 
 func checkErr(err error) {
```

Comment:
sgtm. The scope of this PR is addressing the sever-vs-client error origin (#3937). Are you ok with merging this as is and opening a separate issue for the (mostly mindless) task of converting all commands to use the above error handling pattern?